source/src_io/write_input.cpp

```diff
@@ -226,55 +206,17 @@ void Input::Print(const std::string &fn)const
-	ModuleBase::GlobalFunc::OUTP(ofs,"eels_method",eels_method,"0: hilbert_transform method; 1: standard method");
-	ModuleBase::GlobalFunc::OUTP(ofs,"absorption_method",absorption_method,"0: vasp's method  1: pwscf's method");
-	ModuleBase::GlobalFunc::OUTP(ofs,"system",system_type,"the calculate system");
-	ModuleBase::GlobalFunc::OUTP(ofs,"eta",eta,"eta(Ry)");
-	ModuleBase::GlobalFunc::OUTP(ofs,"domega",domega,"domega(Ry)");
-	ModuleBase::GlobalFunc::OUTP(ofs,"nomega",nomega,"nomega");
-	ModuleBase::GlobalFunc::OUTP(ofs,"ecut_chi",ecut_chi,"the dimension of chi matrix");
-	ofs << std::setw(20) <<"q_start"<<q_start[0]<<"   "
-		<<q_start[1]<<"   "<<q_start[2]
-		<<"  #the position of the first q point in direct coordinate" <<std::endl;
-	ofs << std::setw(20) <<"q_direction"<<q_direct[0]<<"   "<<q_direct[1]<<"   "<<q_direct[2]<<"  #the q direction" <<std::endl;
-	ModuleBase::GlobalFunc::OUTP(ofs,"nq",nq,"the total number of qpoints for calculation");
-	ModuleBase::GlobalFunc::OUTP(ofs,"out_epsilon",out_epsilon,"output epsilon or not");
-	ModuleBase::GlobalFunc::OUTP(ofs,"out_chi",out_chi,"output chi or not");
-	ModuleBase::GlobalFunc::OUTP(ofs,"out_chi0",out_chi0,"output chi0 or not");
-	ModuleBase::GlobalFunc::OUTP(ofs,"fermi_level",fermi_level,"the change of the fermi_level(Ry)");
-	ModuleBase::GlobalFunc::OUTP(ofs,"coulomb_cutoff",coulomb_cutoff," turn on the coulomb_cutoff or not");
-	ModuleBase::GlobalFunc::OUTP(ofs,"kmesh_interpolation",kmesh_interpolation,"calculting <i,0|j,R>");
-	for(int i=0; i<nq; i++)
-	{
-		ofs << std::setw(20) <<"qcar" << qcar[i][0] <<"   "<< qcar[i][1] <<"   "<<qcar[i][2]<<"  #(unit: 2PI/lat0)" << std::endl;
-	}
+	ofs << "\n#Parameters (14.spectrum)" << std::endl;              // pengfei Li add 2016-11-23
 	ModuleBase::GlobalFunc::OUTP(ofs,"ocp",GlobalV::ocp,"change occupation or not");
 	ModuleBase::GlobalFunc::OUTP(ofs,"ocp_set",GlobalV::ocp_set,"set occupation");
-	//ModuleBase::GlobalFunc::OUTP(ofs,"ocp_n",ocp_n,"number of occupation");
-	// for(int i=0; i<ocp_n; i++)
-	// {
-		// ofs << std::setw(20) <<"ocp_kb" << GlobalV::ocp_kb[i]<< std::endl;
-	// }
-	ofs << std::setw(20) <<"lcao_box"<<lcao_box[0]
-		<<"   "<<lcao_box[1]<<"   "
-		<<lcao_box[2]<<"  #the scale for searching the existence of the overlap <i,0|j,R>" <<std::endl;
-	ModuleBase::GlobalFunc::OUTP(ofs," mulliken", GlobalV::mulliken," mulliken  charge or not");//qifeng add 2019/9/10
-	
-	//ModuleBase::GlobalFunc::OUTP(ofs,"epsilon0",epsilon0,"calculate the macroscopic dielectric constant or not");
-	ModuleBase::GlobalFunc::OUTP(ofs,"intrasmear",intrasmear,"Eta");
-	ModuleBase::GlobalFunc::OUTP(ofs,"shift",shift,"shift");
-	ModuleBase::GlobalFunc::OUTP(ofs,"metalcalc",metalcalc,"metal or not");
-	ModuleBase::GlobalFunc::OUTP(ofs,"eps_degauss",eps_degauss,"degauss in calculating epsilon0");
+	ModuleBase::GlobalFunc::OUTP(ofs,"mulliken", GlobalV::mulliken," mulliken  charge or not");//qifeng add 2019/9/10
```

Comment:
ocp, ocp_set and mulliken parameters are not belong to spectrums.

Reply:
So which section do they belong to?
These parameters were placed here before, amd I did not know how to change it.

Reply:
> ocp and ocp_set might related to TDDFT, mulliken is a postprocessing parameter

OK, I get it.
In addition, I plan to place `mulliken`, `noncolin`, `lspinorb` in `Section 1.General`.